sourced/ml/cmd_entries/repos2bow.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from uuid import uuid4
+
+from scipy import sparse
+
+from sourced.ml.extractors import __extractors__
+from sourced.ml.models import BOW, OrderedDocumentFrequencies, DocumentFrequencies
+from sourced.ml.transformers import Engine, UastExtractor, UastDeserializer, Uast2DocFreq, \
+    Uast2TermFreq, HeadFiles, TFIDF
+from sourced.ml.utils import create_engine
+
+
+def repos2bow_entry(args):
+    log = logging.getLogger("repos2bow")
+    engine = create_engine("repos2bow-%s" % uuid4(), **args.__dict__)
+    document_column_name = "repository_id"
```

Comment:
Would be nice to declare a constant somewhere. E.g. new file engine_constants.py or just engine.py which already exists

Reply:
actually, I think to make it as a parameter for cmd entry point. We can choose `document`, `file`, etc. And it is a good idea to make a constant dictionary to map document type to column id.

Reply:
Let's not overcomplicate the configuration, thank god this is not Java. A constant is enough

Reply:
but do we want to use as document only repository?
We need it to use the same code for files as documents

Reply:
I don't see how this question is related to replacing hardcoded strings here with constants from a dedicated file

Reply:
OK, now I see

Reply:
For now - KISS. We will return to this question in the future